ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.tsx

```diff
@@ -474,7 +474,7 @@ const QueryBasic = ({ model, errorListener }: QueryBasicProps) => {
                 }}
               />
             }
-            label="Location"
+            label={TypedMetacardDefs.getAlias({ attr: 'location' })}
```

Comment:
What happens if there is no alias defined for location?

Reply:
In absence of alias, "location" is used as label.